2016-04-02 12:17:33,206 - softwarecenter.ui.gtk3.app - INFO - setting up proxy 'None'
2016-04-02 12:17:35,904 - softwarecenter.region - WARNING - failed to use geoclue: 'org.freedesktop.Geoclue.Error.notAvailable: Geoclue master client has no usable Address providers'
2016-04-02 12:17:36,106 - softwarecenter.backend.reviews - WARNING - Could not get usefulness from server, no username in config file
2016-04-02 12:17:36,122 - softwarecenter.plugin - INFO - activating plugin '<module 'webapps_activation' from '/usr/share/software-center/softwarecenter/plugins/webapps_activation.pyc'>'
2016-04-02 12:17:36,234 - softwarecenter.db.pkginfo_impl.aptcache - INFO - aptcache.open()
2016-04-02 12:17:38,770 - softwarecenter.backend.reviews - WARNING - error creating bsddb: '(22, 'Invalid argument -- BDB0054 illegal flag combination specified to DB_ENV->open')' (corrupted?)
2016-04-02 12:17:38,771 - softwarecenter.backend.reviews - ERROR - trying to repair DB failed
Traceback (most recent call last):
  File "/usr/share/software-center/softwarecenter/backend/reviews/__init__.py", line 358, in _save_review_stats_cache_blocking
    self._dump_bsddbm_for_unity(outfile, outdir)
  File "/usr/share/software-center/softwarecenter/backend/reviews/__init__.py", line 377, in _dump_bsddbm_for_unity
    0600)
DBInvalidArgError: (22, 'Invalid argument -- BDB0054 illegal flag combination specified to DB_ENV->open')
2016-04-02 12:17:44,359 - softwarecenter.ui.gtk3.views.lobbyview - WARNING - skipping exhibit for: 'u'leadwerks\r\nleadwerks-indie'' not available
2016-04-02 12:17:46,565 - softwarecenter.backend.spawn_helper - WARNING - exit code 1 from helper for '['/usr/share/software-center/piston_generic_helper.py', '--datadir', '/usr/share/software-center/', 'SoftwareCenterRecommenderAPI', 'recommend_app', '{"pkgname": "sublime-text"}']'
2016-04-02 12:17:46,565 - softwarecenter.backend.spawn_helper - WARNING - got error from helper: 'WARNING:__main__:404: {'status': '404', 'content-length': '61', 'via': '1.1 sumac.canonical.com:3128 (squid/2.7.STABLE7)', 'x-cache': 'HIT from sumac.canonical.com', 'age': '5', 'strict-transport-security': 'max-age=2592000', 'server': 'Apache', 'etag': '"ac48fe5710ebda334744e2d913e7ba67"', 'cache-control': 'max-age=300', 'date': 'Sat, 02 Apr 2016 09:17:41 GMT', 'content-type': 'text/html; charset=utf-8', 'x-cache-lookup': 'HIT from sumac.canonical.com:3128'}
'
2016-04-02 12:17:46,565 - softwarecenter.db.categories - WARNING - Error while accessing the recommender service: WARNING:__main__:404: {'status': '404', 'content-length': '61', 'via': '1.1 sumac.canonical.com:3128 (squid/2.7.STABLE7)', 'x-cache': 'HIT from sumac.canonical.com', 'age': '5', 'strict-transport-security': 'max-age=2592000', 'server': 'Apache', 'etag': '"ac48fe5710ebda334744e2d913e7ba67"', 'cache-control': 'max-age=300', 'date': 'Sat, 02 Apr 2016 09:17:41 GMT', 'content-type': 'text/html; charset=utf-8', 'x-cache-lookup': 'HIT from sumac.canonical.com:3128'}

2016-04-02 12:17:46,565 - softwarecenter.ui.gtk3.widgets.recommendations - WARNING - Error while accessing the recommender agent: WARNING:__main__:404: {'status': '404', 'content-length': '61', 'via': '1.1 sumac.canonical.com:3128 (squid/2.7.STABLE7)', 'x-cache': 'HIT from sumac.canonical.com', 'age': '5', 'strict-transport-security': 'max-age=2592000', 'server': 'Apache', 'etag': '"ac48fe5710ebda334744e2d913e7ba67"', 'cache-control': 'max-age=300', 'date': 'Sat, 02 Apr 2016 09:17:41 GMT', 'content-type': 'text/html; charset=utf-8', 'x-cache-lookup': 'HIT from sumac.canonical.com:3128'}

2016-04-02 12:17:46,650 - softwarecenter.db.utils - INFO - software-center-agent finished with status 0
2016-04-02 12:17:51,661 - softwarecenter.db.pkginfo_impl.aptcache - ERROR - simulate failed
None
2016-04-02 12:19:21,261 - softwarecenter.db.pkginfo_impl.aptcache - INFO - aptcache.open()
2016-04-02 12:26:30,654 - softwarecenter.ui.gtk3.app - INFO - setting up proxy 'None'
2016-04-02 12:26:31,058 - softwarecenter.backend.reviews - WARNING - Could not get usefulness from server, no username in config file
2016-04-02 12:26:31,060 - softwarecenter.plugin - INFO - activating plugin '<module 'webapps_activation' from '/usr/share/software-center/softwarecenter/plugins/webapps_activation.pyc'>'
2016-04-02 12:26:31,115 - softwarecenter.db.pkginfo_impl.aptcache - INFO - aptcache.open()
2016-04-02 12:26:32,889 - softwarecenter.backend.reviews - WARNING - error creating bsddb: '(22, 'Invalid argument -- BDB0054 illegal flag combination specified to DB_ENV->open')' (corrupted?)
2016-04-02 12:26:32,889 - softwarecenter.backend.reviews - ERROR - trying to repair DB failed
Traceback (most recent call last):
  File "/usr/share/software-center/softwarecenter/backend/reviews/__init__.py", line 358, in _save_review_stats_cache_blocking
    self._dump_bsddbm_for_unity(outfile, outdir)
  File "/usr/share/software-center/softwarecenter/backend/reviews/__init__.py", line 377, in _dump_bsddbm_for_unity
    0600)
DBInvalidArgError: (22, 'Invalid argument -- BDB0054 illegal flag combination specified to DB_ENV->open')
2016-04-02 12:26:33,126 - softwarecenter.ui.gtk3.views.lobbyview - WARNING - skipping exhibit for: 'u'leadwerks\r\nleadwerks-indie'' not available
2016-04-02 12:26:39,516 - softwarecenter.db.utils - INFO - software-center-agent finished with status 0
